packages/tsx-editor/src/transpiler/transpileHelpers.test.ts

```diff
@@ -135,7 +135,7 @@ describe('_supportedPackageToGlobalMap', () => {
       '@fluentui/react': 'FluentUIReact',
       '@fluentui/date-time-utilities': 'FluentUIReact',
       '@fluentui/react-focus': 'FluentUIReact',
-      '@uifabric/foundation': 'FluentUIReact',
+      '@fluentui/foundation-legacy-legacy': 'FluentUIReact',
```

Comment:
```suggestion
      '@fluentui/foundation-legacy': 'FluentUIReact',
```

Reply:
Hmm now I'm really curious if the script did that or if it was some other find/replace... 😬

Reply:
good catch! honestly i am not sure how this happened